repos/system_upgrade/el8toel9/actors/scanblacklistca/libraries/scanblacklistca.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import BlackListCA, BlackListError
+
+# dict(orig_dir: new_dir)
+DIRS_CHANGE = {
+    '/etc/pki/ca-trust/source/blacklist/': '/etc/pki/ca-trust/source/blocklist/',
+    '/usr/share/pki/ca-trust-source/blacklist/': '/usr/share/pki/ca-trust-source/blocklist/'
+}
+
+
+def _get_dirs():
+    return DIRS_CHANGE
+
+
+def _get_files(dirname):
+    # on rhel8, -type can't take two arguments, so we need to call find
+    # twice and concatenate the results
+    files = run(['find', dirname, '-type', 'f'], split=True)['stdout']
+    return files + run(['find', dirname, '-type', 'l'], split=True)['stdout']
+
+
+def _make_messages(dirname, targetname):
```

Comment:
Nit: `_generate_messages` is a better naming fit

Reply:
Ok